rasa_sdk/__main__.py

```diff
@@ -18,20 +22,32 @@ def main_from_args(args):
     )
     utils.update_sanic_log_level()
 
-    run(
-        args.actions,
-        args.port,
-        args.cors,
-        args.ssl_certificate,
-        args.ssl_keyfile,
-        args.ssl_password,
-        args.auto_reload,
-        args.endpoints,
-    )
+    if args.grpc:
+        asyncio.run(
+            run_grpc(
+                args.actions,
+                args.port,
+                args.ssl_certificate,
+                args.ssl_keyfile,
+                args.ssl_ca_file,
+                args.endpoints,
+            )
+        )
+    else:
+        run(
+            args.actions,
+            args.port,
+            args.cors,
+            args.ssl_certificate,
+            args.ssl_keyfile,
+            args.ssl_password,
+            args.auto_reload,
```

Comment:
I see this is missing from run_grpc, do we need to issue a warning in case this gets set when using grpc?

Reply:
Or should we make this feature available with grpc too?